R/args.R

```diff
@@ -31,7 +31,8 @@ CmdStanArgs <- R6::R6Class(
                           refresh = NULL,
                           output_dir = NULL,
                           validate_csv = TRUE,
-                          sig_figs = NULL) {
+                          sig_figs = NULL,
+                          opencl_device = NULL) {
```

Comment:
So the main question is:

`opencl_device = c(1, 0)`

The first number is the platform ID, the other is the device ID. Or do we want a list?

Reply:
Or third option: this is set via option?

Reply:
Of these options, `opencl_device = c(1, 0)` is my favorite.